lib/approvals/writer.rb

```diff
@@ -16,6 +17,7 @@ module Writer
       html: Writers::HtmlWriter.new,
       hash: Writers::HashWriter.new,
       array: Writers::ArrayWriter.new,
+      custom: Writers::CustomWriter.new,
```

Comment:
(cont https://github.com/kytrinyx/approvals/pull/73#discussion_r62084196)

...And then this module could just use the class passed in if `:custom` is set.